google/cloud/speech/connection.py

```diff
@@ -26,7 +26,7 @@ class Connection(base_connection.JSONConnection):
     API_VERSION = 'v1beta1'
     """The version of the API, used in building the API call's URL."""
 
-    API_URL_TEMPLATE = '{api_base_url}/{api_version}/speech:{path}'
+    API_URL_TEMPLATE = '{api_base_url}/{api_version}/{path}'
```

Comment:
To support the `/operations` endpoint.

1. `/speech:syncrecognize`
2. `/speech:asyncrecognize`
3. `/operations`

 Let me know if I should put it back and add logic to handle `operations` somewhere.